introduction_to_amazon_algorithms/imageclassification_mscoco_multi_label/Image-classification-multilabel-lst.ipynb

```diff
@@ -87,7 +87,7 @@
     "\n",
```

Comment:
<p>CI raises a problem here. Don't really know why. You can try to remove <code>%%bash</code>  and use <code>!pip install pycocotools</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
There are a couple of things going on:
1. The methods are for SM <=1 . Does CI is exclusively looking for SM >= 2 ?
2. This line: 

```!wget -O /tmp/test.jpg https://images.pexels.com/photos/763398/pexels-photo-763398.jpeg``` 

breaks because of a failed HTTP request: 

```-2021-01-08 20:50:54--  https://images.pexels.com/photos/763398/pexels-photo-763398.jpeg
Resolving images.pexels.com (images.pexels.com)... 104.17.208.102, 104.17.209.102, 2606:4700::6811:d166, ...
Connecting to images.pexels.com (images.pexels.com)|104.17.208.102|:443... connected.
HTTP request sent, awaiting response... 503 Service Temporarily Unavailable
2021-01-08 20:50:54 ERROR 503: Service Temporarily Unavailable.```